### ISSUE TITLE
How to Upload files and images 

### ISSUE THREAD

**Issue by edzai** [thibaultduponchelle/q2a-wysiwyg-matheditor#1]
Hi, i just downloaded your maths editor. Very good indeed. QUick question, how do I add the upload files and images functionality?

**Comment by thibaultduponchelle** [comment]
Hello, thank you for the kind comment :smiley: 

I don't know, I think I added it in the ckeditor pack (I well see it in the "plugins config").

I have to check, give me some time and I will answer you.

**Comment by edzai** [comment]
Wow, that was a fast response, thank you. Appreciate it.

**Comment by thibaultduponchelle** [comment]
I checked and the "upload image" is not installed/working. We can only embed images already hosted elsewhere (this feature is working).

I checked the wysiwyg editor (in the q2a core), which is based on the same editor framework (ckeditor 4) and the upload is working well.

I then tried to merge wysiwyg editor with my ckeditor build pack to try quick win but it's not working (trivially).

The best solution would be even maybe not to improve my editor but to enhance core plugin to support mathjax (change build pack + add an option), but here also it requires some research and some effort.

(before you probably ask, I don't plan to fix it at the moment, I don't use personally this plugin despite being mine :D).

Regards.

Thibault
